README.md

```diff
@@ -11,66 +11,65 @@ it is runtime-agnostic, but users must implement basic networking logic, chain
+The project implements all of the [BOLT
+specifications](https://github.com/lightning/bolts) in the spec. The
 implementation has pretty good test coverage that is expected to continue to
 improve. It is also anticipated that as developers begin using the API, the
 lessons from that will result in changes to the API, so any developer using this
 API at this stage should be prepared to embrace that. The current state is
-sufficient for a developer or project to experiment with it. Recent increased
-contribution rate to the project is expected to lead to a high quality, stable,
-production-worthy implementation in 2021.
+sufficient for a developer or project to experiment with it.
 
-Communications for Rust-Lightning and Lightning Development Kit happens through our LDK
-[slack](https://join.slack.com/t/lightningdevkit/shared_invite/zt-tte36cb7-r5f41MDn3ObFtDu~N9dCrQ) & [discord](https://discord.gg/5AcknnMfBw) channels.
+Communications for Rust-Lightning and Lightning Development Kit happens through
+our LDK [Discord](https://discord.gg/5AcknnMfBw) channels.
 
 Crates
 -----------
-1. [lightning](./lightning)   
-  The Core of the LDK library, implements the lightning protocol, channel state machine, 
-  and on-chain logic. Supports no-std and exposes on relatively low-level interfaces.
+1. [lightning](./lightning)
+  The core of the LDK library, implements the Lightning protocol, channel state machine,
+  and on-chain logic. Supports `no-std` and exposes only relatively low-level interfaces.
 2. [lightning-background-processor](./lightning-background-processor)
   Utilities to perform required background tasks for Rust Lightning.
 3. [lightning-block-sync](./lightning-block-sync)
   Utilities to fetch the chain data from a block source and feed them into Rust Lightning.
 4. [lightning-invoice](./lightning-invoice)
-  Data structures to parse and serialize BOLT11 lightning invoices.
+  Data structures to parse and serialize
+  [BOLT #11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md)
+  Lightning invoices.
 5. [lightning-net-tokio](./lightning-net-tokio)
-  Implementation of the rust-lightning network stack using Tokio.
-  For Rust-Lightning clients which wish to make direct connections to Lightning P2P nodes,
-  this is a simple alternative to implementing the required network stack, especially for those already using Tokio.
+  Implementation of the rust-lightning network stack using the
+  [Tokio](https://github.com/tokio-rs/tokio) `async` runtime. For Rust-Lightning
+  clients which wish to make direct connections to Lightning P2P nodes, this is
+  a simple alternative to implementing the required network stack, especially
+  for those already using Tokio.
 6. [lightning-persister](./lightning-persister)
-  Utilities to manage Rust-Lightning channel data persistence and retrieval.
+  Implements utilities to manage Rust-Lightning channel data persistence and retrieval.
```

Comment:
```suggestion
  Implements utilities to manage rust-lightning channel data persistence and retrieval.
```

Repo name should probably should be lowercase for consistency too (unless start of a sentence)? Here and elsewhere there's a mix of "Rust-Lightning" and "rust-lightning"

Reply:
Mh, I'm not sure if there is consensus on this. I'd say if we see Rust-Lightning as a thing that is more than the main crates itself, it probably should be capitalized. However, I'd argue `rust-lightning` is just the Rust library and the 'more' part would be LDK and therefore agree to keep it consistently lower case. So I now replaced Rust-Lightning with `rust-lightning` in most places, except for headings and the tag line.

Reply:
Ok, yeah that makes sense. I guess Rust-Lightning was in its previous life :)